src/domain/Loan.ts

```diff
@@ -62,7 +63,15 @@ class Installments {
     this.createdAt = createdAt;
     this.updatedAt = updatedAt;
     this.status = status;
+    this.paidAt = this.paidAt;
```

Comment:
@daniszcode aqui você está atribuindo o this.paidAt a ele mesmo. Ficou uma linha redundante.
Como o paidAt só é atualizado após o status da installment mudar para **paid**, entao no construtor voce nao precisa fazer nada com ele ;)